src/styles/sidebar/components/_index.scss

```diff
@@ -51,3 +51,7 @@
 .js-thread-list-scroll-root {
   overflow-anchor: none;
 }
+
+.break-anywhere {
+  overflow-wrap: anywhere;
+}
```

Comment:
Surprisingly (or not), tailwind does not have a utility covering this case https://tailwindcss.com/docs/word-break, so I had to create it here.

My intention was to extend the theme in tailwind config to add this, but I didn't find a way to do it. I'm open to suggestions if this can be done.